boards/Kconfig

```diff
@@ -3046,315 +3056,261 @@ endif #ARCH_BOARD_CUSTOM
 
 config ARCH_BOARD
 	string
-	default "amber"                     if ARCH_BOARD_AMBER
```

Comment:
> Someone knows what's up with the CI failing?
> I'm not able to decode the errors it's outputting.

@PetervdPerk-NXP 
Most likely this part breaks something. Could you modify this part to be more git-history friendly (restore one space character)?

Reply:
try refresh these configurations:
```
	modified:   boards/arm/imxrt/imxrt1170-evk/configs/can/defconfig
2615
	modified:   boards/arm/imxrt/imxrt1170-evk/configs/canfd/defconfig
2616
	modified:   boards/arm/imxrt/imxrt1170-evk/configs/libcxxtest/defconfig
2617
	modified:   boards/arm/imxrt/imxrt1170-evk/configs/netnsh/defconfig
2618
	modified:   boards/arm/imxrt/imxrt1170-evk/configs/nsh/defconfig
```
	
with `./tools/refresh.sh --silent imxrt1170-evk/can` etc.